conf/application.conf

```diff
@@ -349,6 +349,16 @@ db {
   #default.logSql=true
 }
 
-app.portWebapp = 8085
+pidfile.path = "/dev/null"
```

Comment:
Why?

Reply:
Removed the path reference(No impact on application).